src/components/Header/index.js

```diff
@@ -13,6 +14,9 @@ const Header = ({ className, Dark, Light }) => {
     </svg>
   );
 
+  const Dark = location === routeMap.interested;
```

Comment:
I think you should extract this logic up from the header component. I prefered the Dark / Light props rather than the location prop

Reply:
👍